index.js

```diff
@@ -63,7 +59,15 @@ function parse_desc_stream(descstream, callback) {
 }
 
 function parse_dep(str) {
-  return str.split(/,[\s]*/).map(normalize_ws);
+  return str.split(/,[\s]*/).map(function(str){
+    return str.match(/\(.+\)/) ?
+        {
+            package: str.replace(/\(.+\)/, '').trim(),
+            version: str.replace(/.*\((.+)\)/, '$1').trim()
+        } : {
+            package: str.trim()
+        };
+    });
```

Comment:
Does this handle the case where there are newlines within the dependency, e.g. `dplyr\n(>= 0.5.0)` or `dplyr (>= \n0.5.0)`?

Reply:
Good question :) I'll add a test case and see.